pkg/sentry/csr/csr.go

```diff
@@ -32,17 +30,17 @@ var oidSubjectAlternativeName = asn1.ObjectIdentifier{2, 5, 29, 17}
 func GenerateCSR(org string, pkcs8 bool) ([]byte, []byte, error) {
 	key, err := certs.GenerateECPrivateKey()
 	if err != nil {
-		return nil, nil, errors.Wrap(err, "unable to generate private keys")
+		return nil, nil, fmt.Errorf("unable to generate private keys: %w", err)
 	}
 
 	templ, err := genCSRTemplate(org)
 	if err != nil {
-		return nil, nil, errors.Wrap(err, "error generating csr template")
+		return nil, nil, fmt.Errorf("error generating csr template: %w", err)
 	}
 
 	csrBytes, err := x509.CreateCertificateRequest(rand.Reader, templ, key)
 	if err != nil {
-		return nil, nil, errors.Wrap(err, "failed to create CSR")
+		return nil, nil, fmt.Errorf("failed to create CSR: %w", err)
```

Comment:
nit: `csr` instead of `CSR`?

Reply:
CSR is an acronym (Certificate Signing Request) so normally uppercase is correct.